process/Github code review.md

```diff
@@ -0,0 +1,38 @@
+# GitHub command line code review
+
+There are two options to start a code review on Github:
+
+## Option 1: Use the review.py script from l2tdevtools:
```

Comment:
I would change the order of the options starting with the Github native one first